src/java.base/share/lib/security/default.policy

```diff
@@ -154,6 +154,9 @@ grant codeBase "jrt:/jdk.crypto.cryptoki" {
                    "accessClassInPackage.sun.security.*";
     permission java.lang.RuntimePermission "accessClassInPackage.sun.nio.ch";
     permission java.lang.RuntimePermission "loadLibrary.j2pkcs11";
+    permission java.lang.RuntimePermission
+                    "accessClassInPackage.openj9.internal.security";
+    permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
```

Comment:
This wasn't required for jdk11: it should not be required here either.

Reply:
updated